<ui version="4.0" >
 <class>cs8ModbusWidgetBase</class>
 <widget class="QWidget" name="cs8ModbusWidgetBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>277</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <layout class="QGridLayout" >
     <item row="0" column="0" >
      <widget class="QLabel" name="label" >
       <property name="text" >
        <string>Name</string>
       </property>
       <property name="buddy" >
        <cstring>leName</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1" >
      <widget class="QLineEdit" name="leName" >
       <property name="text" >
        <string>Modbus</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2" >
      <widget class="QLabel" name="label_2" >
       <property name="text" >
        <string>Port</string>
       </property>
       <property name="buddy" >
        <cstring>spPort</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="3" >
      <widget class="QSpinBox" name="spPort" >
       <property name="minimum" >
        <number>1</number>
       </property>
       <property name="maximum" >
        <number>65535</number>
       </property>
       <property name="value" >
        <number>502</number>
       </property>
      </widget>
     </item>
     <item row="1" column="2" >
      <widget class="QLabel" name="label_3" >
       <property name="text" >
        <string>Connections</string>
       </property>
       <property name="buddy" >
        <cstring>spConnections</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="3" >
      <widget class="QSpinBox" name="spConnections" >
       <property name="minimum" >
        <number>1</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableView" name="tableView" >
     <property name="editTriggers" >
      <set>QAbstractItemView::DoubleClicked|QAbstractItemView::EditKeyPressed|QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="dragEnabled" >
      <bool>true</bool>
     </property>
     <property name="alternatingRowColors" >
      <bool>true</bool>
     </property>
     <property name="selectionMode" >
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior" >
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="gridStyle" >
      <enum>Qt::NoPen</enum>
     </property>
     <property name="wordWrap" >
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
  <action name="actionInsert_Item" >
   <property name="text" >
    <string>Insert Item</string>
   </property>
  </action>
  <action name="actionDelete_Item" >
   <property name="text" >
    <string>Delete Item</string>
   </property>
  </action>
  <action name="actionAppend_Item" >
   <property name="text" >
    <string>Append Item</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
